cmd/version.go

```diff
@@ -14,7 +14,7 @@ See the License for the specific language governing permissions and
 limitations under the License.
 */
 
-package main
+package cmd
```

Comment:
Seems to version getting form the `info.Main.Version` should be adopted here like in the controller-tools project https://github.com/kubernetes-sigs/controller-tools/blob/main/pkg/version/version.go

Otherwise, the version printed after the `go install ...` will be always `unknown`.

Reply:
I'm not sure if Kubebuilder itself tracks the same version as controller runtime. That might be a change in behaviour. If one of the maintainers wants to make that change, it's fine for me.

Reply:
I'm talking about controller-tools version, I just say about approach to get a `kubeBuilderVersion` value from the the binary `info.Main.Version` value for the kubebuilder project itself.

Reply:
@migueleliasweb can you please address this one ?

Reply:
I'not sure I fully understand what needs to be done here. Using `debug.ReadBuildInfo()` might yield a change in behaviour compared to the current setup.

Would it be okay to address this in a separate PR? (I'm just trying to make this one specific for the `go-installable` feature for kubebuilder.

If we still want to move ahead, I'm happy to mimic the same logic as in https://github.com/kubernetes-sigs/controller-tools/blob/main/pkg/version/version.go.

Reply:
Is this what we're looking for?

If the `kubeBuilderVersion` is not set, fallback to `info.Main.Version`.

```go
func versionString() string {
	if kubeBuilderVersion == "" {
		info, ok := debug.ReadBuildInfo()
		if ok || info.Main.Version != "" {
			kubeBuilderVersion = info.Main.Version
		}
	}

	return fmt.Sprintf("Version: %#v", version{
		kubeBuilderVersion,
		kubernetesVendorVersion,
		gitCommit,
		buildDate,
		goos,
		goarch,
	})
}
```

Reply:
Pushed a change about this. Let me know if it looks good so I can squash everything before the merge. :saluting_face: